setup.cfg

```diff
@@ -11,7 +11,7 @@ packages=['upcloud_api', 'upcloud_api.cloud_manager']
 license = MIT
 
 [options]
-python_requires = >=3.7, <4
+python_requires = >=3.8, <4
```

Comment:
Is this because 3.7 is EOL or are we using features introduced in 3.8? Could also update this later if we do not want to block 3.7 users from installing the package yet 🤔

Reply:
For example https://github.com/UpCloudLtd/upcloud-python-api/pull/143 seems to fail on 3.7, as might other future package upgrades. 3.7 is also EOL.